api/v1alpha1/dspipeline_types.go

```diff
@@ -194,6 +194,9 @@ type MariaDB struct {
 	// Customize the size of the PVC created for the default MariaDB instance. Default: 10Gi
 	// +kubebuilder:default:="10Gi"
 	PVCSize resource.Quantity `json:"pvcSize,omitempty"`
+	// volumeMode Filesystem storageClass to use for PVC creation
```

Comment:
this is trivial but the description provided here very slightly differs from what is committed in the CRD below. (volumeMode vs Volume Mode).  They are functionally the same but a side effect of this is that `make manifests` will update the CRD which may confuse some.

So with that said, could you reconcile this please (either change this to "Volume Mode" or update `config/crd/bases/datasciencepipelinesapplications.opendatahub.io_datasciencepipelinesapplications.yaml` and run `make manifests`? to ensure they match up?